main/src/library/MutList.flix

```diff
@@ -566,19 +566,20 @@ namespace MutList {
             ()
 
     ///
-    /// Inserts all the elements in the given mutable list `v2` into the mutable list `v1`.
-    ///
-    pub def append!(v1: MutList[a], v2: MutList[a]): Unit & Impure =
-        let MutList(a1, l1) = v1;
-        let MutList(a2, l2) = v2;
-        let len1 = deref l1;
-        let len2 = deref l2;
-        if (len2 > Array.length(deref a1) - len1)
-            reserve!(len2, v1)
-        else
-            ();
-        Array.updateSequence!(len1, deref a2, deref a1);
-        l1 := len1 + len2
+    /// Inserts all the elements in the collection `t` into the mutable list `v`.
+    ///
+    pub def append!(v: MutList[a], t: t[a]): Unit & Impure with Iterable[t] =
```

Comment:
Should `v` be the last argument?

Reply:
Maybe not sure, can you open a new ticket?

Reply:
Yes, I will do that